tests/Rector/Contrib/Symfony/Controller/AddFlashRector/AddFlashRectorTest.php

```diff
@@ -0,0 +1,36 @@
+<?php declare(strict_types=1);
+
+namespace Rector\Tests\Rector\Contrib\Symfony\Controller\RedirectToRouteRector;
```

Comment:
Namespace wrong. Should be `Rector\Tests\Rector\Contrib\Symfony\Controller\AddFlashRector`